src/subworkflows/subworkflow.js

```diff
@@ -90,7 +90,9 @@ export class Subworkflow extends BaseSubworkflow {
             _id: Cls.generateSubworkflowId(),
             name,
             application: application.toJSON(),
-            properties: lodash.sortedUniq(lodash.flatten(units.map((x) => x.resultNames))),
+            properties: lodash.sortedUniq(
+                lodash.flatten(units.filter((x) => x.resultNames).map((x) => x.resultNames)),
```

Comment:
Nit, but it would be a bit more understandable if we do map first and then filter out null values with `filter((x) => x)`, no?